smartsim/_core/_cli/utils.py

```diff
@@ -33,18 +34,21 @@
 from smartsim._core.utils import colorize
 from smartsim.log import get_logger
 
-smart_logger_format = "[%(name)s] %(levelname)s %(message)s"
-logger = get_logger("Smart", fmt=smart_logger_format)
+SMART_LOGGER_FORMAT = "[%(name)s] %(levelname)s %(message)s"
+logger = get_logger("Smart", fmt=SMART_LOGGER_FORMAT)
 
 
 def get_install_path() -> Path:
-    try:
-        import smartsim as _
-    except (ImportError, ModuleNotFoundError):
+    module_spec = importlib.util.find_spec("smartsim")
+
+    if module_spec is None:
         raise SetupError("Could not import SmartSim") from None
 
     # find the path to the setup script
-    package_path = Path(_.__path__[0]).resolve()
+    package_dir = ""
+    if module_spec.submodule_search_locations:
+        package_dir = module_spec.submodule_search_locations[0]
+    package_path = Path(package_dir).resolve()
```

Comment:
I'm having a lot of difficulty seeing whether this code is equivalent to what it's replacing. Need to have other reviewers check it more carefully.

Reply:
<ditto above>